configure.ac

```diff
@@ -1954,6 +1967,11 @@ fi
 case $add_players in
   *omxplayer*)
       XB_ADD_PLAYER([OMXPLAYER], [omxplayer])
+      AC_DEFINE([TARGET_RASPBERRY_PI],[1],["Define to 1 for RASPBERRY_PI"])
```

Comment:
Without that line If I only use the "omxplayer" options kodi does not build if the TARGET_RASPBERRY_PI is not defined.
Maybe we can fix the compilation issue ...